node/src/main/scala/org/bitcoins/node/SpvNode.scala

```diff
@@ -30,7 +30,11 @@ case class SpvNode(
 
   override val peer: Peer = nodePeer
 
-  override val callbacks: SpvNodeCallbacks = nodeCallbacks
+  override def setCallbacks(callbacks: NodeCallbacks): Node =
```

Comment:
is this kind of dangerous? It replaces all existing call backs right? maybe we just want a `addCallback()`? I would at least add a scaladoc saying it removes existing callbacks and replaces them with the given ones.